src/pages/AnswerSurvey.js

```diff
@@ -0,0 +1,76 @@
+import React, { useContext, useEffect, useState } from 'react';
+import { useParams, useNavigate } from "react-router-dom";
+import {
+  IconButton,
+  Typography,
+  Button,
+  Box
+} from "@mui/material";
+import AddIcon from '@mui/icons-material/Add';
+import ArrowBackIcon from '@mui/icons-material/ArrowBack';
+
+import RestClient from '../utils/RestClient';
+import AppContext from '../contexts/AppContext';
+import NewQuestionDialog from '../components/NewQuestionDialog';
+import Question from '../components/Question';
+
+const EditSurvey = () => {
+  const appContext = useContext(AppContext);
+  const params = useParams();
+  const navigate = useNavigate();
+  const [survey, setSurvey] = useState(null)
+  const [answer, setAnswer] = useState([]);
+  useEffect(() => {
+    // Fetch the survey on load
+    fetchSurvey()
```

Comment:
This won't work for users that are not authenticated.

Reply:
We might want to revisit who needs  to be authenticated in the future. We can take a look later- not critical for showing being able to answer surveys

Reply:
It *is* critical for answering surveys, the primary use case for answering surveys is unauthenticated users.

Reply:
We can revisit in Milestone 3

Reply:
Im going to add a backend change that allows non-authenticated users to answer a survey